### PR TITLE
Cleaned up tank sprites, and updates inhands

### LINKED ISSUE #9116: Tank sprite inhands don't match the tank sprites
These inhands still haven't been updated.

image

image

### PR DESCRIPTION
**Pull request by Partheo**
Cleaned up the jetpack and air tank sprites, and made the classic jetpacks use an overlay when on instead of having a different sprite.

The air tank inhands have been updated, they now properly reflect the tank sprites we have (rather than the TG ones we had prior).

Fixes  #9116

#### Changelog

:cl:  Partheo
imageadd: New inhand sprites for air tanks
/:cl:

### CODE REVIEW

**Comment by fluffe9911** [comment]
nice

File: code/game/objects/items/tanks/jetpack.dm [hunk 1]
```diff
@@ -51,16 +52,26 @@
...
+	update_icon()
 	ion_trail.start()
 	RegisterSignal(user, COMSIG_MOVABLE_MOVED, .proc/move_react)
 	if(full_speed)
 		user.add_movespeed_modifier(MOVESPEED_ID_JETPACK, priority=100, multiplicative_slowdown=-2, movetypes=FLOATING, conflict=MOVE_CONFLICT_JETPACK)
 
+/obj/item/tank/jetpack/update_icon()
+	icon_state = initial(icon_state)
+	if(!classic) //does the jetpack have its own on sprite?
+		if(on)
```
Comment (JamieD1) [inline code/game/objects/items/tanks/jetpack.dm]:
```suggestion
	if(!classic && on) //does the jetpack have its own on sprite?
```

**Review by JamieD1 — CHANGES_REQUESTED** [review]
Minor change, but then good to go

File: code/game/objects/items/tanks/jetpack.dm [hunk 2]
```diff
@@ -51,16 +52,25 @@
...
+	update_icon()
 	ion_trail.start()
 	RegisterSignal(user, COMSIG_MOVABLE_MOVED, .proc/move_react)
 	if(full_speed)
 		user.add_movespeed_modifier(MOVESPEED_ID_JETPACK, priority=100, multiplicative_slowdown=-2, movetypes=FLOATING, conflict=MOVE_CONFLICT_JETPACK)
 
+/obj/item/tank/jetpack/update_icon()
+	icon_state = initial(icon_state)
+	if(!classic && on) //does the jetpack have its own on sprite?
+			icon_state = "[initial(icon_state)]-on"
```
Comment (JamieD1) [inline code/game/objects/items/tanks/jetpack.dm]:
```suggestion
		icon_state = "[initial(icon_state)]-on"
```

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**